datasets/imagenet.py

```diff
@@ -48,7 +48,6 @@ def imagenet_get_datasets(data, load_train=True, load_test=True,
                 transforms.RandomResizedCrop(input_size, antialias=True),
                 transforms.RandomHorizontalFlip(),
                 transforms.ToTensor(),
-                transforms.Normalize((0.485, 0.456, 0.406), (0.229, 0.224, 0.225)),
                 ai8x.normalize(args=args),
```

Comment:
PyTorch suggests these normalisation coefficients -as far as I remember-, is there a specific reason for removing those?

Reply:
These are indeed the suggested coefficients for Imagenet. However, as these coefficients include small 
standard deviation values it causes most of the inputs to be clamped at ai8x.normalize() stage.

Reply:
Sorry, I later remembered I had also removed those in my local experiments for latter use of ai8x normalize that assumes interval is 0-1. (more proper normalization can be clipping values and arranging range but removing is also fine for now)

Reply:
Great catch btw, we had to remove those for Bayerization studies as well

Reply:
Thanks! The good thing is as Bayerization was using these augmentation settings as well, with this change we removed it from the Bayerization dataset as well 😄